### PR TITLE
Pull request by student-team. Update project with a base student views

### PR DESCRIPTION
**Pull request by MaryanDev**

### CODE REVIEW

File: eQuiz/modules/eQuiz.Web/Areas/Student/Controllers/DefaultController.cs [hunk 1]
```diff
@@ -13,5 +13,20 @@ public ActionResult Index()
         {
             return View();
         }
+
+        public ActionResult Details()
+        {
+            return View();
+        }
+
+        public ActionResult Login()
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Controllers/DefaultController.cs:22]: Why do we need the "Login" action here?

File: eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Details.cshtml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+﻿
+@{
+    Layout = null;
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Details.cshtml]: I'd suggest to use https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Views/Shared/Layout.cshtml as a layout.

File: eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Dashboard.cshtml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+﻿
+@{
+    Layout = null;
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Dashboard.cshtml]: I'd suggest to use https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Views/Shared/Layout.cshtml as a layout.

File: eQuiz/modules/eQuiz.Web/Areas/Student/Styles/Dashboard.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#headerTable {
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Styles/Dashboard.css]: Let's use only one css file: https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Areas/Student/Styles/equiz-temp.css.

File: eQuiz/modules/eQuiz.Web/Areas/Student/Styles/login.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
+#loginContent {
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Styles/login.css]: Let's use only one css file: https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Areas/Student/Styles/equiz-temp.css.

File: eQuiz/modules/eQuiz.Web/Areas/Student/Styles/studentDetails.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
+h1 {
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Styles/studentDetails.css]: Let's use only one css file: https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Areas/Student/Styles/equiz-temp.css.

File: eQuiz/modules/eQuiz.Web/Controllers/HomeController.cs [hunk 7]
```diff
@@ -13,5 +13,10 @@ public ActionResult Index()
         {
             return RedirectToAction("Index", "Account");
         }
+
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Controllers/HomeController.cs:16]: What should the "MainLogin" action do?

File: eQuiz/modules/eQuiz.Web/Views/Home/MainLogin.cshtml [hunk 8]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Views/Home/MainLogin.cshtml:1]: In my opinion current view just duplicates the https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Views/Account/Index.cshtml.

Is it correct?

File: eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Details.cshtml (hunk 2 above)
Comment (MaryanDev) [inline eQuiz/modules/eQuiz.Web/Areas/Student/Views/Default/Details.cshtml]: This layout contains header which is unnecessary in other views.

File: eQuiz/modules/eQuiz.Web/Views/Home/MainLogin.cshtml (hunk 8 above)
Comment (MaryanDev) [inline eQuiz/modules/eQuiz.Web/Views/Home/MainLogin.cshtml:1]: This view was created according to wireframes

**Pull request closed without merging**